_docs/vision.md

```diff
@@ -15,39 +15,39 @@ PostgreSQL?
 
 Welcome to Babelfish. The idea of the project is to close the gap between
 PostgreSQL and other systems by making PostgreSQL speak other protocols such as
-TDS, and maybe a lot more in the future. By implementing "MS SQL"-slang, we make
```

Comment:
Second sentence suggestion (more active):

Babelfish closes the gap between PostgreSQL and Microsoft SQL Server by teaching PostgreSQL to speak TDS protocol.

Reply:
Done in 5baa4f274d0f91f4f79426a2913dfc1c804ca521.
I left the original version without focus on SQL Server, because the vision of the project is to implement other protocols as well.

Reply:
The vision of the project is not to support other protocols as well. Perhaps that may happen, but it is not the vision.

Reply:
I see; I had thought Amazon to be more visionary.
I have changed the paragraph in 1c64123b4c06915555ed28fe99ac484e979ac19e; I hope that is better. I had to modify the suggestion somewhat for accuracy.